product_docs/docs/pgd/5/cli/command_ref/index.mdx

```diff
@@ -1,10 +1,10 @@
 ---
 title: Command reference
 navTitle: Command reference
-description: A reference guide to the commands available in the (PGD CLI).
+description: A reference guide to the commands available in the PGD CLI.
 ---
 
-pgd is the command name for the PGD command line interface.
+The command name for the PGD command line interface is `pgd`.
```

Comment:
Made this change so as not to start with lower case command, but then I think we need to tag it in any case.